MAUI/Calendar/views.md

```diff
@@ -0,0 +1,190 @@
+                        <Calendar:CalendarWeekNumberStyle Background="DeepSkyBlue">
+                            <Calendar:CalendarWeekNumberStyle.TextStyle>
+                                <Calendar:CalendarTextStyle TextColor="White" FontSize="12" />
+                            </Calendar:CalendarWeekNumberStyle.TextStyle>
+                        </Calendar:CalendarWeekNumberStyle>
+                    </Calendar:CalendarMonthView.WeekNumberStyle>
+                </Calendar:CalendarMonthView>
+            </Calendar:SfCalendar.MonthView>
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.MonthView = new CalendarMonthView()
+{
+    ShowWeekNumber = true,
+};
+
+CalendarTextStyle textStyle = new CalendarTextStyle()
+{
+    TextColor = Colors.Black,
+    FontSize = 12,
+};
+
+this.Calendar.MonthView.WeekNumberStyle = new CalendarWeekNumberStyle()
+{
+    Background = Colors.DeepSkyBlue,
+    TextStyle = textStyle,
+};
+
+{% endhighlight %}
+{% endtabs %}
+
+![week-number-style-in-maui-calendar](images/views/week-number-style-in-maui-calendar.png)
+
+
+## Year view
+The Year view displays the current year's month. A calendar year is a one-year period that begins on January 1 and ends on December 31. By default, initially displays the current years month and the current month is highlighted by a separate color different from the rest of the month color in `Year` view. Can easily navigate to the desired month dates from the year view.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                        View="Year">
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.View = CalendarView.Year;
+
+{% endhighlight %}
+{% endtabs %}
+
+![Year view in .NET MAUI Calendar.](images/views/maui-year-view.png)
+
+## Decade view
+The Decade view displays the period of ten years and some years of next view. By default, initially displays the current year view and the current year is highlighted by a separate color different from the rest of the years color in `Decade` view. Can easily navigate to the desired year in the Year view from the decade view.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                        View="Decade">
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.View = CalendarView.Deacde;
+
+{% endhighlight %}
+{% endtabs %}
+
+![Decade view in .NET MAUI Calendar.](images/views/maui-decade-view.png)
+
+## Century view
+The Century view displays the period of hundred years and some years of next view. By default, initially displays the current range of years and the current year range is highlighted by a separate color different from the rest of the years color in `Century` view. Can easily navigate to the Decade view from the Century view.
```

Comment:
The Century view displays the period of hundred years and some years ahead. By default, displays the current range of years, and the current year range is highlighted by a separate color different from the rest of the years' color in the `Century` view. You can easily navigate to the Decade view from the Century view.

Reply:
Content changed for the century view.